data/scripts/valadoc.babel.js

```diff
@@ -0,0 +1,171 @@
+// Some configuration that shouldn't be modified at runtime.
```

Comment:
This file should also load pages like the previous script did. When a user clicks a link in the same domain we fetch the HTML of that link, replacing the current content area with the new content area, but not touching the sidebar. This has the benefit of being faster, and not changing the sidebar when searching.